### ISSUE TITLE
doc-review spellcheck highlights words as mis-spelt even when in known_words 

### ISSUE THREAD

**Issue by ds-jdcockrill** [demisto/demisto-sdk#1978]
SDK version: 1.6.2

### Reproduction steps

* following the [Adopt-a-Pack instructions](https://xsoar.pan.dev/docs/partners/adopt#text-for-the-pack) I created a new release note using the command: `demisto-sdk update-release-notes -i Packs/DigitalShadows -f`
* I updated the generated release note to replace the placeholder text with `- Completed Adopt-a-Pack process (https://xsoar.pan.dev/docs/partners/adopt#process).`
* I ran the `doc-review` command: `demisto-sdk doc-review -i Packs/DigitalShadows/ReleaseNotes/1_0_3.md`
* I see some warnings, including:

```
 - Words that might be misspelled were found in Packs/DigitalShadows/ReleaseNotes/1_0_3.md:
  - Adopt-a-Pack - did you mean: ['ca', 'co', 'ci', 'cki', 'cu']
If these are not misspelled consider adding them to a known_words file:
  Pack related words: content/Packs/<PackName>/.pack-ignore under the [known_words] section.
  Not pack specific words: content/Tests/known_words.txt
To test locally add --use-packs-known-words or --known-words flags.
```

* I add 'Adopt-a-Pack' to either the `.pack-ignore` or `content/Tests/known_words.txt` and re-run with the appropriate switches.
* It still tells me that `Adopt-a-Pack` is misspelled.

Note: the same behaviour seems to occur with words like "IoC" (note the lowercase middle letter).

### Expected behaviour

* if I add 'Adopt-a-Pack' to either of the `known_words` mechanisms, I shouldn't be warned about it anymore.
* perhaps misspellings shouldn't be highlighted red by the SDK and instead just shown as warnings.

**Comment by ds-jdcockrill** [comment]
See attached file for full log during the `doc-review` command.

[demisto-sdk-issue-1978_1.log](https://github.com/demisto/demisto-sdk/files/8324601/demisto-sdk-issue-1978_1.log)

**Comment by dorschw** [comment]
Thanks for opening an issue, we'll take a look and update here / close this issue with a fix PR.

**Comment by dorschw** [comment]
@ds-jdcockrill this fix will be available starting on demisto-sdk v1.6.4, planned to be released later this month.
